compiler/src/main/kotlin/visitors/prettyprinter.kt

```diff
@@ -58,7 +58,7 @@ class PrettyPrinter : BaseASTVisitor() {
                 "${node.dimensions[n].size} ${if (node.dimensions[n].type == WRAPPING) {
                     "[wrap]$separator"
                 } else if (node.dimensions[n].type == EDGE) { // Note the non-null assertion, required for field access
-                    "[edge=${node.dimensions[n].edge!!.spelling}]$separator"
+                    "[edge$separator"
```

Comment:
```suggestion
                    "[edge]$separator"
```

Missing a closing square bracket

Reply:
Fixed in 958249b